src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/Models/CreateOrganizationUser.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Models;
+
+public class CreateOrganizationUser
```

Comment:
Add some xmldoc here or maybe rename it to something that would indicate what this is used for.

Reply:
xmldoc still required